src/App.jsx

```diff
@@ -1,104 +1,28 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
```

Comment:
Ensure that the file './api/comments.json' exists and is correctly formatted as a JSON file. If there are any issues with the file path or format, it could cause errors when importing data.